internal/cli/config/config.go

```diff
@@ -85,17 +96,39 @@ Enter [?] on any option to get help.
 
 `, config.ToolName)
 
-	q := opts.accessQuestions()
+	q := accessQuestions(opts.IsOpsManager())
 	if err := survey.Ask(q, opts); err != nil {
 		return err
 	}
+	opts.SetUpAccess()
 
-	q = opts.defaultQuestions()
-	if err := survey.Ask(q, opts); err != nil {
+	if err := opts.initStore(); err != nil {
 		return err
 	}
 
-	if err := opts.Save(); err != nil {
+	if config.IsAccessSet() {
+		var err error
+
+		opts.OrgID, err = opts.askOrg()
```

Comment:
[np] if we are setting a property of `opts` should we just do this in the function call to begin with?

Reply:
fair point, let me check how this looks